boards/FSM/Src/main.c

```diff
@@ -170,11 +174,16 @@ int main(void)
     Io_SharedHardFaultHandler_Init();
 
     Io_FlowMeters_Init(&htim4);
-    primary_flow_meter = App_FlowMeter_Create(Io_FlowMeters_GetPrimaryFlowRate);
-    secondary_flow_meter =
-        App_FlowMeter_Create(Io_FlowMeters_GetSecondaryFlowRate);
+    primary_flow_meter =
```

Comment:
Minimum flow rate threshold set as 1 L/min for now
https://www.adafruit.com/product/828